test/App.test.tsx

```diff
@@ -0,0 +1,189 @@
+import { describe, it, expect } from 'vitest'
```

Comment:
split this into a test for each component separately
BasicSide.test.ts, BasicHead.test.ts, etc

Reply:
Ah okay - as opposed to
```
describe('BasicSideTest', () => {
  it('render-basic-side', () => {
    render(<BasicSide />)
  })
  it('get-basic-side', () => {
    // check if variable correct
  })
}

describe('BasicHeadTest', () => {
  it('render-basic-head', () => {
    render(<BasicHead />)
  })
  it('get-basic-head', () => {
    // check if variable correct
  })
}
```
?

Reply:
Ah okay - as opposed to
```
describe('BasicSideTest', () => {
  it('render-basic-side', () => {
    render(<BasicSide />)
  })
  it('get-basic-side', () => {
    // check if variable correct
  })
}

describe('BasicHeadTest', () => {
  it('render-basic-head', () => {
    render(<BasicHead />)
  })
  it('get-basic-head', () => {
    // check if variable correct
  })
}
```
?